nucleus/validate/scenario_test.py

```diff
@@ -170,6 +171,12 @@ def get_items(self) -> List[DatasetItem]:
             DatasetItem.from_json(item) for item in response[DATASET_ITEMS_KEY]
         ]
 
+    def get_scenes(self) -> List[Scene]:
+        response = self.connection.get(
+            f"validate/scenario_test/{self.id}/scenes",
+        )
+        return [Scene.from_json(scene) for scene in response[SCENES_KEY]]
+
```

Comment:
I don't like exposing this dichotomy to the end user. Since tests can really only be of one level we should keep the same interface for all items of a test.